drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInBottomSheetDialogFragment.kt

```diff
@@ -81,6 +82,11 @@ internal abstract class DropInBottomSheetDialogFragment : BottomSheetDialogFragm
         return false
     }
 
+    override fun onCancel(dialog: DialogInterface) {
+        Logger.d(TAG, "onCancel")
```

Comment:
What do you think of keeping `super.onCancel(dialog)` call here, in case the logic of `onCancel()` changes in `DialogFragment`?
Though I am not sure if it would add any value.

Reply:
If they would change the super implementation it would most likely only have unwanted side effects, so I think it's better to leave it out